src/en/index.md

```diff
@@ -83,33 +83,31 @@ was stated earlier still applies. For instance, if Juju removes a machine then
 MAAS will, in turn, release that machine to the pool.
 
 
-## A simple MAAS setup
+## Key components and colocation of all services
 
-The key components of a MAAS installation are:
+The key components of a MAAS installation are the region controller and the
+rack controller. See [Concepts and terms](intro-concepts.html#controllers) for
+how each are defined.
 
-- Region controller(s)
-- Rack controller(s)
-- MAAS machines (nodes)
+Unless there is specific reason not to, it is recommended to have both
+controllers residing on the same system. A no-fuss way to achieve this is by
+installing the `maas` metapackage (or while installing Ubuntu Server with an
+ISO).
```

Comment:
The Ubuntu Server bit doesn't quite make sense to me, perhaps:
A no-fuss way to achieve this is by installing the `maas` metapackage, or by installing from the Ubuntu Server ISO.